### PR TITLE
FEATURE: rlc_component without name

### PR DESCRIPTION
**Pull request by gkorompi**

### CODE REVIEW

File: tests/legacy/system/test_edb_modeler.py
Comment (SMoraisAnsys) [inline tests/legacy/system/test_edb_modeler.py:469]: Any reason for such change ?

Comment (gkorompi) [reply tests/legacy/system/test_edb_modeler.py:469]: instead of writing a new test i just added this line, which tests the changes.

**Review by SMoraisAnsys — DISMISSED** [review]
LGTM

File: tests/legacy/system/test_edb_modeler.py
Comment (SMoraisAnsys) [reply tests/legacy/system/test_edb_modeler.py:470]: I see that now the two lines are kept so my previous comment does not apply any more.

**Comment by SMoraisAnsys** [comment]
@gkorompi Please avoid using auto merge because I can't accept a review otherwise (my approval would make the PR getting merge). I prefer that either @ring630 or @svandenb-dev also give their opinion before merging something as they are the core maintainers.

**Review by ring630 — APPROVED** [review]
LGTM

**Pull request merged**
